kratos/includes/model_part.h

```diff
@@ -1532,27 +1545,27 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     ///@name Member Variables
     ///@{
 
-    std::string mName;
+    std::string mName; /// The name of the model part
 
-    IndexType mBufferSize;
+    IndexType mBufferSize; /// The buffers size of the database
 
-    ProcessInfo::Pointer mpProcessInfo;
+    ProcessInfo::Pointer mpProcessInfo; /// The process info instance
 
-    TablesContainerType mTables;
+    TablesContainerType mTables; /// The tables contained on the model part
 
-    std::vector<IndexType> mIndices;
+    std::vector<IndexType> mIndices; /// The list of indices /// NOTE: This looks unused, only appears on the constructor @pooyan_dadvand
```

Comment:
ping @pooyan-dadvand

Reply:
Historical residuals. AKA to be removed....